### ISSUE TITLE
Don't add derived column for group by and order by if item and select ailas match

### ISSUE THREAD

**Issue by terrymanu** [shardingjdbc/sharding-jdbc#646]
for example: 
The SQL like 

```sql
SELECT o.user_id uid FROM t_order o GROUP BY o.user_id ORDER BY o.user_id
```

should not add derived column, sharding-jdbc should match alias and item.

**Comment by terrymanu** [comment]
reported by #645

**Comment by terrymanu** [comment]
fixed at 2.1.0
